.github/workflows/pr.yml

```diff
@@ -0,0 +1,17 @@
+name: Validate PRs
+
+on:
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  docker:
+    name: Check docker build
+    runs-on: ubuntu-latest
```

Comment:
FYI I've noticed that we used that all over the place but we should stick to a given version
like ubuntu-20.04 (or ubuntu-18.04 as you used latest) to avoid surprise when `latest` alias is changing

Reply:
Good catch! I'll update it here and I will keep it in mind for further my PRs.